languages/python/Assignments/assignment1.md

```diff
@@ -0,0 +1,42 @@
+# Python Assignments
+
+## Assignment 1
+
+### Level 0
+
+1. Write a programm for fibonacci series in python.
+2. Write a programm for printing the star partterns:
+
+```
+(A)      (B)      (C)       (D)
+*       *****    *****         *
+**       ****    ****         **
+***       ***    ***         ***
+****       **    **         ****
+*****       *    *         *****
+
+    (E)
+     *
+    * *
+   * * *
+  * * * *
+ * * * * *
+  * * * *
+   * * *
+    * *
+     *
+```
+
+3. WAP for no of occurrences of the characters in the string. Print it’s count and index.
```

Comment:
The majority of the people will not I believe. cc @akoserwal @Jaanki @rahulbajaj0509